breacharbiter.go

```diff
@@ -580,7 +579,7 @@ secondLevelCheck:
 	if err != nil {
 		brarLog.Errorf("unable to broadcast "+
 			"justice tx: %v", err)
-		if strings.Contains(err.Error(), "already been spent") {
+		if err == lnwallet.ErrDoubleSpend {
```

Comment:
@cfromknecht Can you make sure this check does what it is intended to do now?

Reply:
If it is supposed to go to `secondLevelCheck` in the `ErrDoubleSpend` case. (NOTE: this error is not returned in case the _same_ tx is published, only if the output was already spent by a conflicting transaction)

Reply:
Also, maybe a comment explaining why it dose to `secondLevelCheck` in this case would be useful :p

Reply:
The goal of this check is that if we try to broadcast the justice transaction, but one or some of the outputs have already been spent, then we'll get an error. We'll then try to advance the first non-second level output to the second level and then try again. So the error that we need to catch here is double spend or already spent, etc.